### ISSUE TITLE
[BUG] ND freezing after changing approach

### ISSUE THREAD

**Issue by FilbertFlies** [flybywiresim/a32nx#2488]
**Mod Version**

  "built": "2020-12-17T07:38:03+00:00",
  "ref": "",
  "sha": "",
  "actor": "Andreas Guther <andreas@guther.family>",
  "event_name": "manual"

**Describe the bug**

After selecting arrival procedure in the MCDU en-route, the ND freezes and won't unfreeze. The aircraft appears stationary on the ND, and changing the mode or range has no effect. This happens simultaneously on both captain's and FO's NDs.

**To Reproduce**

1. While en-route, select a new runway, STAR & transition for approach.
2.
3.

**Expected behavior**

I expected the ND to update with the new arrival routing, and for it to continue to display my progress as normal.

**Actual behavior**

The aircraft's position on the ND froze. Initially, the distance to go and ETE fields remained stalled, then later disappeared.

**References**

https://youtu.be/E6KamikqWnQ

**Additional context**

Was this working before/when did the issue start occurring?

The issue first occurred yesterday in a previous version of the custom FBW master build.

Is this a problem in the vanilla unmodded game?

No

Discord username (if different from GitHub):

FilbertFlies#3763

**Comment by ThatRedMelon** [comment]
Do you have the WorkingTitle CJ4 beta installed? If so try downgrading to stable, it seems there's a conflict currently.

**Comment by FilbertFlies** [comment]
No, I have the stable version installed.

**Comment by ThatRedMelon** [comment]
Could you try without and see if the issue persists?

**Comment by FilbertFlies** [comment]
Certainly. I'll give it a go in the morning.

**Comment by FilbertFlies** [comment]
1) I decided I'd give it a go now, for the sake of a better night's sleep!
2) I was totally wrong. I did have the beta version installed, I just didn't know it was a beta version.
3) Yes, deleting it seems to have solved the problem. I haven't done a full flight, but I've taken off and changed the approach multiple times, and all seems to be well.

Thank you very much for the super-quick response and resolution. Thanks also for the fantastic work you're all doing with this aircraft.
